src/dune_rules/merlin_ident.ml

```diff
@@ -3,19 +3,25 @@ open Import
 type t =
   | Lib of Lib_name.t
   | Exes of string list
+  | Melange_entries of string list
```

Comment:
Should just be `string`

Reply:
https://github.com/ocaml/dune/pull/6454/commits/0b7f398c2184e5109de9055007cf502c9f356198